docs/examples/viz_pbr_interactive.py

```diff
@@ -0,0 +1,353 @@
+"""
+===============================================
+Interactive PBR demo
+===============================================
+
+This is a demonstration of how Physically-Based Rendering (PBR) can be used to
+simulate different materials.
+
+Let's start by importing the necessary modules:
+"""
+
+from fury import actor, material, ui, window
+from fury.data import fetch_viz_cubemaps, read_viz_cubemap
+from fury.io import load_cubemap_texture
+from fury.utils import (normals_from_actor, tangents_to_actor,
+                        tangents_from_direction_of_anisotropy)
+
+
+"""
+This function will help us to define the appearance of our labels.
+"""
+
+
+def build_label(text, font_size=16, color=(1, 1, 1), bold=False, italic=False,
+                shadow=False):
+    label = ui.TextBlock2D()
+    label.message = text
+    label.font_size = font_size
+    label.font_family = 'Arial'
+    label.justification = 'left'
+    label.bold = bold
+    label.italic = italic
+    label.shadow = shadow
+    label.actor.GetTextProperty().SetBackgroundColor(0, 0, 0)
```

Comment:
Can you add this in TextBlock2D as options? Can be on another PR.

Reply:
Sorry, I just noticed this is already in the UI component.